### DISCUSSION TITLE
what to do after removing a cluster from data

### DISCUSSION THREAD

**Discussion by meroosh** [satijalab/seurat#6607] [Q&A]
After I  integrated my data, and decided to remove a cluster from my data, what should my next step be?
Should I look again for variable features? re-scale? re-normalize? before clustering again?

**Comment by atly2000** [comment]
I am not sure what your goal is for this data, but you could take the remaining barcodes and use those to create a new Seurat object before clustering again.
When you create your new Seurat object you can specify the barcodes you want to keep.
